frontend/tests/App.test.tsx

```diff
@@ -1,16 +0,0 @@
-import * as React from 'react';
```

Comment:
I think this is a dummy test from the template. It seems like we're putting tests under /src instead of under /tests so I removed this one. Let me know if it should stick around